prow/plugins/plugins.go

```diff
@@ -191,6 +208,21 @@ func (c *Configuration) TriggerFor(org, repo string) *Trigger {
 	return nil
 }
 
+func (c *Configuration) setDefaults() {
+	for _, config := range c.Plugins {
```

Comment:
Rather than only defaulting if the plugin is used, why not just say
```go
if c.ConfigUpdater.ConfigFile == "" {
	c.ConfigUpdater.ConfigFile = "prow/config.yaml"
}
if c.ConfigUpdater.PluginFile == "" {
	c.ConfigUpdater.PluginFile = "prow/plugins.yaml"
}
```
?

Reply:
Yeah, we could do that, too. Updated.